apps/backend/src/lambdas/pos-reconciliation.ts

```diff
@@ -0,0 +1,40 @@
+import { format, parse, subBusinessDays } from 'date-fns';
+import { DateRange, Ministries, NormalizedLocation } from '../constants';
+import { POSDepositEntity } from '../deposits/entities/pos-deposit.entity';
+import { PosReconciliationService } from '../reconciliation/pos-reconciliation.service';
+import { PaymentEntity } from '../transaction/entities';
+
+export const posReconciliationHandler = async (
```

Comment:
Organization-wise, does this make sense in the lambdas section, or should it be maybe in a service?

Reply:
resolved?